Makefile

```diff
@@ -233,8 +233,7 @@ $(JSONNET_FMT): vendor |  $(BIN_DIR)
 	go build -mod=vendor -o $@ github.com/google/go-jsonnet/cmd/jsonnetfmt
 
 $(KUBEVAL): $(BIN_DIR)
-	go get -d github.com/instrumenta/kubeval
-	go build -o $@ github.com/instrumenta/kubeval
+	go build -mod=vendor -o $@ github.com/instrumenta/kubeval
```

Comment:
lgtm but let's put this in separate PRs next time to keep things clean. There is no need to rush things through :+1: